src/test/java/com/checkmarx/flow/cucumber/integration/cli/sast/SastCliSteps.java

```diff
@@ -100,6 +105,37 @@ public void runningWithBreakBuild(String issueType) {
         testContext.setCxFlowExecutionException(exception);
     }
 
+    @When("bug tracker is set to 'None'")
+    public void setNoneBugTracker() {
+        flowProperties.setBugTracker(BugTracker.Type.NONE.toString());
+    }
+
+    @Then("running cxflow to execute SAST scan")
+    public void runCxFlowScan() {
+        Throwable exception = null;
+        try {
+            TestUtils.runCxFlow(testContext.getCxFlowRunner(), STANDARD_CLI_SCAN + GITHUB_REPO_ARGS + TEST_NONE_BT_PROJECT_NAME);
+        } catch (Throwable e) {
+            exception = e;
+        }
+        testContext.setCxFlowExecutionException(exception);
+    }
+
+    @Then("cxflow should not wait for scan results")
+    public void validateScanNotFinished() {
+        Integer projectId = cxService.getProjectId("CxServer/SP/Checkmarx", TEST_NONE_BT_PROJECT_NAME);
+
+        log.info("Checking if there is any existing scan for Project:");
+        Integer existingScanId = cxService.getScanIdOfExistingScanIfExists(projectId);
+
+        Assert.assertNotEquals(-1, Optional.ofNullable(existingScanId));
```

Comment:
Code analysis says: 
> Possible redundant assertion: incompatible types are compared 'int' and 'Optional<Integer>'

So this assertion will never fail.

Reply:
Good catch!
fixed and verified